### PR TITLE
Cybran projectiles

### PR DESCRIPTION
**Pull request by rajderks**
Updated all (probably) TM Cybran units:
- Avalance had an effect a few times bigger than its radius.. lolz
- T2/T3/T4 Rockets are now normalized.
- Hippo/Dervish cannon explosions were way out there. They have been normalized
- Removed a bunch of unused projectiles
- Removed 2 completely unused files
- Adjusted Mayhem to not use Cybran rockets, but Rather the Blood Asps ones;
- Toned down the green laser on Eliash (thing was thick.)
- Lowered the volume on the T1 Improved PD (slyfox?) as it was louder than a battleship.
- Immortal. Now actually works. Set rocket arc to low, uses T2 rockets, NotExclusive added.
- Hippo: NotExclusive added. 
- All gauss cannon from the mechwarrios have been toned down.
- Some random things.

Basically the whole Cybran lineup was the most offensive one. If you spawn all the TM Experimentals and agronar, dervish, hippo and immortal you will see the obvious difference in screen clutter being cut.

Let me know if needs adjustments.

### CODE REVIEW

**Comment by rajderks** [comment]
Forgot to mention. All the gauss cannons have been toned down too
